### ISSUE TITLE
Teach browsertime to collect Telemetry

### ISSUE THREAD

**Issue by ncalexan** [mozilla/browsertime#9]
It will be helpful if browsertime would "just go and get" Telemetry collected during a test run.  
Certain performance measurements that could be helpful are captured in Telemetry.  In addition, attaching the "environment" to a browsertime report could be really helpful in identifying the browser and hardware characteristics of the machine running the test.

One technical direction this could take is to add browser-specific measurements and/or privileged JS measurements.  I started working on a general framework for such privileged JS measurements but don't have a link at the moment.  Alternatively, this could be a new browsertime feature, which might help extract the files from Android devices.

**Comment by bdekoz** [comment]
yeah, I have integrated your work and extended it to collect scalars, histograms, environment

See: this branch
https://github.com/bdekoz/browsertime/tree/privileged-telemetry

And then what populates the files are these scripts:

Here is what I use to dump environment
https://github.com/bdekoz/browsertime/blob/privileged-telemetry/browserscripts/vendor/privileged_telemetry_environment.js


Here is what I use to dump histograms
https://github.com/bdekoz/browsertime/blob/privileged-telemetry/browserscripts/vendor/privileged_telemetry_histograms.js


Here is what I use to dump scalars
https://github.com/bdekoz/browsertime/blob/privileged-telemetry/browserscripts/vendor/privileged_telemetry_scalars.js
